api/src/main/java/org/openmrs/module/allergyapi/AllergenType.java

```diff
@@ -14,5 +14,5 @@
 package org.openmrs.module.allergyapi;
 
 public enum AllergenType {
-	DRUG, FOOD, ENVIRONMENT
+	DRUG, FOOD, ENVIRONMENT, OTHER
```

Comment:
@dkayiwa  As per Darius's comment on https://github.com/openmrs/openmrs-core/pull/2028 ,
Core should support DRUG, FOOD, ENVIRONMENT, OTHER although only 3 are being used. Similarly, I made it such that even allergyapi also supports all the 4.